react-components/src/hooks/useGroundPlaneFromScene.tsx

```diff
@@ -77,13 +77,24 @@ export const useGroundPlaneFromScene = (sceneExternalId: string, sceneSpaceId: s
     }
     const groundMeshes: CustomObject[] = [];
 
+    // New scale format was introduced as a bug in the Scenebuilder
+    // on August 1st, 2024. This is a temporary fix to handle both formats.
+    // This should be removed when we upgrade the Scene Data Model to a new version
+    const groundPlaneUsesNewScaleFormat = groundPlaneUseNewScaleFormat(
+      scene.sceneConfiguration.updatedAt
```

Comment:
Remember to write this down in your document, so that we remember to actually remove this :+1:

Reply:
Agreed!